### DISCUSSION TITLE
Почему так мало языков для перевода с русского языка?

### DISCUSSION THREAD

**Discussion by kaiso77** [ilyhalight/voice-over-translation#528] [Q&A] [answered]
раньше был немецкий, испанский и многие другие, сейчас только английский, почему сократили?

**Comment by NullVerdict** [comment, marked as answer]
> раньше был немецкий, испанский и многие другие, сейчас только английский, почему сократили?

В шапке всё написано:

Большая часть языков доступных для выбора в качестве озвучки были отключены на стороне **Яндекса**
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/19c9319d-f4a4-49d0-9700-94405c4bdad8)

**Comment by Prottok** [comment]
Я хочу сказать важную вещь. Что в версий плагина 1.4.1 перевод на других языках РАБОТАЕТ. Например это ролик на немецком свободно переводится на русский (https://www.youtube.com/watch?v=pgUTqx0qUQY), также и с испанским языком итд. Я не знаю откуда эта взялось мол переводчик не работает на других языках кроме англ, когда это не так, по крайней мере у меня. Верните пожалуйста функцию перевода с других языков.

**Reply by FiLinX** [reply]
не про то думаешь..  
НА РУССКИЙ - яндыкс то, наверно не менее чем с 99 языков переводит
а НА ДРУГИЕ для озвучки - сейчас только на те три (раньше много было)
и да, работает и с русского на русский )

**Comment by YuchkinUA** [comment]
@kaiso77 @Prottok Раньше по моему даже и Украинский был ( какое-то время, если не ошибаюсь) 
но увы. (то что в Яндекс переводчике до сих пор есть Украинский язык, опустим)

Конешно, можно попробовать установить старые версии но... а толку?
<img width="1280" height="640" alt="image" src="https://github.com/user-attachments/assets/40bab6f8-266c-4bfe-a058-f69942b1df88" />

Если и добавлять новые, то это надо всё переписовать по новой - а это морока.

p.s. в моем случае это версия 1.3.5

**Reply by ilyhalight** [reply]
> Раньше по моему даже и Украинский был ( какое-то время, если не ошибаюсь)
> но увы. (то что в Яндекс переводчике до сих пор есть Украинский язык, опустим)

это именно Яндекс ограничил озвучиваемые языки до этих 3-х. Раньше, и правда, там было гораздо больше языков, но сейчас даже, если передать любой из старых языков вернется ошибка от сервера
<img width="580" height="122" alt="WindowsTerminal_ZF8B1AExhZ" src="https://github.com/user-attachments/assets/f680a5cf-a533-438a-88ff-03f8a1b34e65" />

**Comment by YuchkinUA** [comment]
<img width="1280" height="762" alt="image" src="https://github.com/user-attachments/assets/d2ede7c2-d4d6-42a1-9cc7-c55c10c49571" />

@ilyhalight I might not be the first.

On the same topic, is it possible to add real-time translation in the future?

Thank.

**Reply by ilyhalight** [reply]
> On the same topic, is it possible to add real-time translation in the future?

idk tbh, stay tuned
